packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -116,21 +113,17 @@ export const start = async () => {
         WrappedApp: routeComponent(App, false, locals)
     }
 
-    await loadableReady()
-
-    if (queryParams.has('__client_only')) {
-        window.__HYDRATING__ = false
-        const root = createRoot(rootEl)
-        root.render(<OuterApp {...props} />)
-    } else {
-        hydrateRoot(
-            rootEl,
-            <OuterApp
-                {...props}
-                onHydrate={() => {
-                    window.__HYDRATING__ = false
-                }}
-            />
-        )
-    }
+    return Promise.resolve()
```

Comment:
Changes in this file are simply for reverting the old `__client_only` logic. See commit 50cbe7d6576b75f0126e911b8da3d24732ff0bfa